_maps/asteroidstation.dm

```diff
@@ -17,12 +17,14 @@ Names of z-level do not matter, but order does greatly, for instances such as ch
         #include "map_files\AsteroidStation\z5.dmm"
         #include "map_files\generic\z6.dmm"
         #include "map_files\generic\z7.dmm"
+		#include "map_files\generic\z8.dmm"
```

Comment:
They match up in the viewer, because the old lines were actually just spaced out to that length.

That wasn't obvious to me until I saw it in this viewer. In any case it works either way.